tracing-macros/examples/factorial.rs

```diff
@@ -18,6 +18,7 @@ fn factorial(n: u32) -> u32 {
 
 fn main() {
     env_logger::Builder::new().parse("trace").init();
+    #[allow(deprecated)]
```

Comment:
I added this so that *all* warnings were silenced since I was running clippy with `cargo clippy --frozen --all-targets -- -D warnings` and needed to ensure that the warnings didn't get swallowed up on subsequent runs (there were originally around 100 warnings). I'd fix it properly if I knew how.